components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.html

```diff
@@ -15,12 +15,12 @@
   <chef-click-outside (clickOutside)="closeDropdown()">
     <chef-dropdown [attr.visible]="active">
       <chef-checkbox
-        *ngFor="let project of projectsArray()"
+        *ngFor="let project of projectsArray"
         [checked]="project.checked"
         [attr.title]="project.name"
         (change)="projectChecked($event.detail, project)"
-        (keydown.enter)="closeColumnDropdown()"
-        (keydown.esc)="closeColumnDropdown()"
+        (keydown.enter)="closeDropdown()"
+        (keydown.esc)="closeDropdown()"
```

Comment:
PR extra: bug fix! This function had been renamed some time ago but these were missed.